README.md

```diff
@@ -1,2 +1,150 @@
+
+## Post-Build Action
+
+The plugin needs the **Archive the artifacts** post-build action. The
+regeneration of trends could be triggered. Please archive the artifact
+before Refresh trends.
+
+![](/docs/images/post-build-actions.png)
+
+**Example Trend Graphs**
+
+**![](/docs/images/graphs.png)}**
+
+## Pipelines Steps
+
+The "neoloadRun" step in the Jenkins Snippet Generator makes it possible
+to run a NeoLoad scenario from Jenkins. It also archives the reports and
+refreshes the graphs.
+
+-   Warning: To use the Snippet Generator, the Jenkins project including
+    the job to configure must be compliant with Pipeline as code. For
+    more information, see [Pipeline as
+    code](https://jenkins.io/doc/book/pipeline-as-code/#introduction).
+
+Once the Jenkins project is selected, the **Snippet Generator** is
+accessible with a click on the **Pipeline Syntax** link.
+
+This plugin provides two steps: 
+
+-   neoloadRun: to run NeoLoad scenario, archive report and refresh the
+    trends.
+
+-   neoloadRefreshTrends: to refresh or change the trends only.
+
+##### Execute NeoLoad
+
+```groovy
+neoloadRun executable: '/opt/neoload/bin/NeoLoadCmd', project: 'test.nlp', scenario: 'Test for CD', trendGraphs: ['AvgResponseTime', 'ErrorRate']
+neoloadRun executable: '/opt/neoload/bin/NeoloadCmd', project: [server: 'NTS', name: 'MyProject', publishTestResult: false], scenario: 'Test for CD', trendGraphs: ['AvgResponseTime', 'ErrorRate']
+neoloadRun executable: '/opt/neoload/bin/NeoloadCmd', project: 'test.nlp', scenario: 'Test for CD', sharedLicense: [server: 'NTS', duration: 2, vuCount: 50], trendGraphs: ['AvgResponseTime', 'ErrorRate']
+```
+
+#####  Refresh graph
+
+```groovy
+neoloadRefreshTrends(trendGraphs: ['AvgResponseTime', 'ErrorRate'])
+neoloadRefreshTrends(trendGraphs: [[name: 'Cpu vs User Load', curve: ['Controller/User Load', 'LG localhost:7100/CPU Load'], statistic: 'error'], 'AvgResponseTime', 'ErrorRate'])
+```
+
+## FAQ
+
+Why don't I see any trend graphs?
+
+In order to see trend graphs, please verify:
+
+-   The **Archive the artifacts** post-build action has been added.
+-   Either **Default Report File Names** or **Custom Report File
+    Names** is selected and an xml report is defined.
+-   At least two executions were run.
+-   Date and time is synchronized between the Jenkins machine and the
+    build machine.
+
+## Known Issues
+
+1. The NeoLoad report file (via artifacts and the Performance Result
+link) displays a blank page. This affects versions released before
+NeoLoad 5.2.
+
+#### Workaround
+
+Use the Jenkins Script Console to disable the sandboxing security by
+executing the following script. The Script Console is under Jenkins -\>
+Manage Jenkins -\> Script Console.
+
+    System.setProperty("hudson.model.DirectoryBrowserSupport.CSP", "")
+
+Clear the cache afterwards (hold shift and reload the page).
+
+See
+<https://wiki.jenkins-ci.org/display/JENKINS/Configuring+Content+Security+Policy>
```

Comment:
It will be good to set here an analog page in jenkins.io/doc/... domain and create new PR for migration info from [this wiki-page](https://wiki.jenkins-ci.org/display/JENKINS/Configuring+Content+Security+Policy) to analog page.